### PR TITLE
Reset of the config.yml file and replacement of template variables

### PR DESCRIPTION
**Pull request by kyle-nevins**
This PR contains the reset of the config.yml file to its state in commit 210bd345c37178dae5e646bd636a4718efd3892f and replacing the template variables with the actuals. 

Once this PR is approved we will execute another PR request to the parent repo and work with Pivotal to push it up. 

**Added**
None

**Removed**
- Removed the template variables from config.yml by resetting to the previous version

**Notes**
The reason for this is because Pivotal has a centralized config.yml file that they use for all customers. Modification of the config.yml is not allowed. 
I only committed the docs-content changes as well as the config.yml reset for testing. 

**Testing**
- Follow instructions documented in the Readme to serve docs locally
- Review resulting page on localhost:4567 for any factual, typographical, or grammatical errors.

### CODE REVIEW

File: docs-content/feature-overview.html.md.erb [hunk 1]
```diff
@@ -3,20 +3,23 @@ title: Feature Overview of Crunchy PostgreSQL for PCF
...
 ###<a id='plansize'></a> Plan Sizes
 
 The plans that are included are as follows:
 
-<% vars.service_plans.each do |service| -%>
-- <%= service %>
-<% end -%>
+<ul>
+  <li>small</li>
+  <li>medium</li>
```
Comment (tomswartz07) [inline docs-content/feature-overview.html.md.erb]: File should not use HTML format, use Markdown list type instead.

```
- small
- medium
- large
- extra-large
```

File: docs-content/using.html.md.erb [hunk 2]
```diff
@@ -19,11 +19,14 @@ This service includes:
...
 ###<a id='service-info'></a> PostgreSQL Plans
-The `<%= vars.service_name -%>` service comes with several plans:
+The `postgresql-9.5-odb` service comes with several plans:
 
-<% vars.service_plans.each do |service| -%>
-- <%= service %>
-<% end -%>
+<ul>
+  <li>small</li>
+  <li>medium</li>
```
Comment (tomswartz07) [inline docs-content/using.html.md.erb]: same markdown issue.

File: docs-content/_product_snapshot.erb [hunk 3]
```diff
@@ -29,4 +29,3 @@ Crunchy PostgreSQL for PCF:
         <td>AWS, Azure, GCP, OpenStack, and vSphere</td>
     </tr>
 </table>
-
```
Comment (tomswartz07) [inline docs-content/_product_snapshot.erb]: Can you ensure that the lack of newline at end of this partial does not affect the layout of the sections?
I've noted that some of the partials need the trailing newlines in order to ensure they do not 'bump up' against the next section.

Comment (kyle-nevins) [reply docs-content/_product_snapshot.erb]: Looks ok from my local check out. Table has some space before the LIMITATIONS header.

**Comment by kyle-nevins** [comment]
Changed the HTML to Markdown per Tom's notes.

File: docs-content/_product_snapshot.erb (hunk 3 above)
Comment (tomswartz07) [reply docs-content/_product_snapshot.erb]: Good by me. 👍

**Pull request merged**